src/pivot-table/components/cells/styles.tsx

```diff
@@ -68,3 +68,34 @@ export const StyledLabel = styled("div", {
   fontSize,
   fontFamily,
 }));
+
+export const AdjusterHitArea = styled(Box, {
+  shouldForwardProp: (prop: string) => prop !== "isLastColumn",
+})(({ isLastColumn = false }: { isLastColumn: boolean }) => ({
+  display: "flex",
+  position: "absolute",
+  height: "100%",
+  top: 0,
+  left: `100%`,
+  cursor: "col-resize",
+  // last column padding, other double padding + border
+  width: `${isLastColumn ? 4 : 9}px`,
+  justifyContent: isLastColumn ? "flex-end" : "center",
+  "&&:hover:not(:focus, :active)": {
+    "& .sn-pivot-table-column-adjuster-border": {
+      background: "#D9D9D9",
```

Comment:
`background` probably needs to be a bit more dynamic. As some cell might have this exact color or something other color which will give low contrast.

Reply:
true, you have logic for that somewhere right?

Reply:
Thinking some more about it. When the dimension has background color by expression, the color can be different for each cell. That case is probably not worth trying to handling, but if the cell is getting its color from the `StyleService`, it's going to be the same for all cells. In that case case you could use the `getHoverColor()` from the utils repo to get a contrasting color.

Reply:
But I'd say lets handle that special case in a different PR

Reply:
yeah exactly, this component need to contrast two different backgrounds potentially. I'll write it down